doc/overview.rst

```diff
@@ -29,6 +29,13 @@ The development of PyGMT has been supported by NSF grants
 `OCE-1558403 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1558403>`__ and
 `EAR-1948603 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1948602>`__.
 
+    We welcome any feedback and ideas!
```

Comment:
I think you should either format this as a note, by adding the `.. note::` directive before the paragraph, or remove the indentation to be at the same level as the other text in this section.